pkg/tfbridge/info.go

```diff
@@ -121,6 +122,38 @@ type ProviderInfo struct {
 	SchemaPostProcessor func(spec *pschema.PackageSpec)
 }
 
+// Send logs or status logs to the user.
+//
+// Logged messages are pre-associated with the resource they are called from.
+type Logger interface {
```

Comment:
Could we use the existing interface from the Go SDK here?
 https://github.com/pulumi/pulumi/blob/ee25d63598eae99b4bc170d7b0595fd8609abf50/sdk/go/pulumi/log.go#L29-#L29 

What is Status()? Is it the same as LogArgs.Ephemeral = true?

Reply:
We can't use that interface because it expects `Resource`s, not `URN`s. It's at the wrong level.

`Status()` is the same `LogArgs.Ephemeral = true`. The normal `provider.Host.Log` has `Debug`, `DebugStatus`, etc arguments. That's where I took the name from.

Reply:
Ah yes. Well there's something to be said about making codebases more independent by not borrowing interfaces anyway. Thanks!